### PR TITLE
Adjust setup

### PR DESCRIPTION
**Pull request by mbucknell**
Before making a pull request
----------------------------

- [X] Make sure all tests run
- [ ] Update the changelog appropriately

Description
-----------
Added new make targets to assist in setting up and running in the local development environment. Introduced the python-dotenv module to provide a way to easily set environment variables in the local development environment from a .env file.

After making a pull request
---------------------------
- [ ] If appropriate, put the link to the PR in the JIRA ticket
- [ ] Assign someone to review unless the change is trivial

### CODE REVIEW

File: wellregistry/postgres/migrations/0001_create_db_users.py [hunk 1]
```diff
@@ -12,7 +12,7 @@
 
 from django.db import migrations
 from django.conf import settings
-import postgres.pgsql_utils as pgsql
+import wellregistry.pgsql_utils as pgsql
```
Comment (mbucknell) [inline wellregistry/postgres/migrations/0001_create_db_users.py:15]: I moved this out of postgres app as it seemed like more generally reusable  methods. By moving it out the registry app is not dependent on the postgres app either which may be a good thing.

File: wellregistry/wellregistry/settings.py [hunk 2]
```diff
@@ -26,20 +24,10 @@
...
+SECRET_KEY = os.getenv('SECRET_KEY', get_random_secret_key())
 
 # SECURITY WARNING: don't run with debug turned on in production!
-debug_setting = os.getenv('DEBUG', 'False')
-try:
-    DEBUG = bool(strtobool(debug_setting))
-except ValueError:
-    DEBUG = False
-
-logging.basicConfig(format="%(asctime)s [%(levelname)-8s] %(message)s",
```
Comment (mbucknell) [inline wellregistry/wellregistry/settings.py:41]: I moved the configuration of the root logger to the LOGGING settings below.

**Comment by mbucknell** [comment]
At the moment codacy still fails on the failed imports of third-party dependencies and the name of the migration modules. I've tracked down the problem to the fact that codacy does not know how to load our dependencies so I have sent them a question about this. I have also disabled several of the tools under Prospector in Codacy. This is how  have set up waterdataui and by using PyLint we are meeting PEP 257.

**Comment by mbucknell** [comment]
From Codacy, the can't find import error is a false positive. They do their analysis in a sandbox without connecting to the internet so they have no way of installing third party dependencies. However, you can ignore the issue: https://support.codacy.com/hc/en-us/articles/360009180134-The-Issues-View#2-ignoring-issue which I have done with my PR.

File: wellregistry/wellregistry/settings.py [hunk 3]
```diff
@@ -26,20 +24,10 @@
 # See https://docs.djangoproject.com/en/3.0/howto/deployment/checklist/
 
 # SECURITY WARNING: keep the secret key used in production secret!
-try:
-    from .local_settings import SECRET_KEY
-except ImportError:
-    SECRET_KEY = os.getenv('SECRET_KEY', get_random_secret_key())
+SECRET_KEY = os.getenv('SECRET_KEY', get_random_secret_key())
```
Comment (ayan-usgs) [inline wellregistry/wellregistry/settings.py:27]: This is so much better than what I had originally put here.

**Pull request merged**